build/fetch-material-icons.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+set -e
+
+REPO_URL="https://github.com/Templarian/MaterialDesign"
+CLONE_PATH="/tmp/material-design-icons"
+TARGET_PATH="$PWD/img/material"
+
+if [ ! -d "${CLONE_PATH}" ]; then
+    git clone "${REPO_URL}" --depth 1 "${CLONE_PATH}"
+fi
+
+mkdir -p "$TARGET_PATH"
+
+cp $CLONE_PATH/LICENSE $TARGET_PATH/LICENSE
+
+cat $CLONE_PATH/meta.json | 
+    jq '.[]|select(.author == "Google" and (.name | contains("-") | not))' > $TARGET_PATH/meta.json
```

Comment:
Fine for simplicity, but want to point out that some icons are unique while having a dash, for instance `cash-100`, `air-purifier` or `contactless-payment-circle`.

Reply:
Yeah we can extend the list later on